public/resources/js/directives/copy.js

```diff
@@ -25,7 +25,7 @@ angularIO.directive('copyContainer', function() {
     transclude: true,
     template:
       '<div style="position: relative">' +
-        '<copy-button style="position: absolute; top: 10px; right: 0px; z-index: 1000" ></copy-button>' +
+        '<copy-button style="position: absolute; top: 10px; right: 0px; z-index: 4" ></copy-button>' +
```

Comment:
This is not enough. Actually, there’s no reason for the z-index to be larger than 1, so you should set it to this exact value.